Testing/Unit/Python/sitkImageTests.py

```diff
@@ -467,6 +467,21 @@ def test_masked_assign_constant(self):
             self.assertEqual(image[0,1], 2)
             self.assertEqual(image[1,1], 0)
 
+
+        for pixel_type in (sitk.sitkVectorFloat32, sitk.sitkVectorFloat64,
+                           sitk.sitkVectorUInt8, sitk.sitkVectorInt8,
+                           sitk.sitkVectorUInt16, sitk.sitkVectorInt16,
+                           sitk.sitkVectorUInt32, sitk.sitkVectorInt32,
+                           sitk.sitkVectorUInt64, sitk.sitkVectorInt64, ):
+            image = sitk.Image([2, 2], pixel_type)
+
+            image[mask_image] = 2
+            self.assertTrue(all(i == 0 for i in image[0,0]))
+            self.assertTrue(all(i == 2 for i in image[1,0]), image[1,0])
```

Comment:
Statement looks strange, not sure if this is correct.

Reply:
The second argument is a message. In this case it would just be the actual value. It is not consistent with the other vector comparisons. But it is OK.